react/src/components/ResourceAllocationFormItems.tsx

```diff
@@ -113,7 +110,11 @@ const ResourceAllocationFormItems: React.FC<
   const [isPendingAgentList, startAgentListTransition] = useTransition();
 
   const currentProject = useCurrentProjectValue();
-  const currentResourceGroup = useCurrentResourceGroupValue(); // use global state
+  const currentResourceGroupInFrom =
```

Comment:
```suggestion
  const currentResourceGroupInForm =
```

Reply:
I'm not sure if this is the intended logic, but backend.ai currently allows an empty string for the resource group name.

Reply:
Thank you for reporting. I fixed it.